holoviews/operation/element.py

```diff
@@ -430,11 +431,11 @@ def _process(self, element, key=None):
         else:
             contour_fn = plt.contour
             contour_type = Contours
+
         if type(element) is Raster:
             data = [np.flipud(element.data)]
-        elif isinstance(element, Raster):
-            data = [element.data]
-        
+        elif isinstance(element, Image):
+            data = [np.flipud(element.dimension_values(2, flat=False))]
         elif isinstance(element, QuadMesh):
             data = (element.dimension_values(0, False),
                     element.dimension_values(1, False),
```

Comment:
They are tested via that gif in the one of notebook tests but thinking about it I suppose the operation is symmetric. But apparently I also added some unit tests for these.